tests/playwright/pages/wp-admin-page.js

```diff
@@ -31,7 +31,7 @@ exports.WpAdminPage = class wpAdminPage {
 		try {
 			await this.page.click( 'text=Create New Page', { timeout: 5000 } );
 		} catch ( err ) {
-			console.error( "Click on Elementor 'Create New Page' button failed" );
+			// console.error( "Click on Elementor 'Create New Page' button failed" );
```

Comment:
Don't do this - deal with the problem itself - make sure you are in the dashboard before clicking "Create New Page"

Reply:
I changed the function completely